pwa/src/templates/documentation/contactDocumentation/ContactDocumentationTemplate.tsx

```diff
@@ -1,19 +1,19 @@
 import * as React from "react";
 import * as styles from "./ContactDocumentationTemplate.module.css";
 import { Container } from "@conduction/components";
-import { Heading1, LeadParagraph, Link } from "@gemeente-denhaag/components-react";
 import { navigate } from "gatsby";
 import { ExternalLinkIcon } from "@gemeente-denhaag/icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faEnvelope, faPhone } from "@fortawesome/free-solid-svg-icons";
+import { Document, Heading1, Paragraph, Link } from "@utrecht/component-library-react";
```

Comment:
Since you have changed to import of the `Link` component the icons do not work anymore.
The icons need to be show to the user so the user knows if the link will redirect them to en External page or an Internal page.
You can show to icon by putting the icon unfront of the text:  
```
<Link>
    <ExternalLinkIcon /> Conduction
</Link>
```
Make sure that the icon is inline with the text.

Reply:
The links also do not have a hover state anymore.